WL4EditorWindow.cpp

```diff
@@ -1068,32 +1079,45 @@ bool WL4EditorWindow::UnsavedChangesPrompt(QString str)
 /// Clear eventhing in the current room.
 /// But at least one door will be kept.
 /// </summary>
-void WL4EditorWindow::CurrentRoomClearEverything()
+/// <param name="no_warning">
+/// Optional param for showing warning of deleting doors.
+/// </param>
+/// <param name="roomId">
```

Comment:
roomId is not needed here. The only usage of roomId here is replacing selectedRoom, and the only time that ClearEverythingInRoom(bool, int) is called is after setting selected room to this value.

Reply:
Seems resolved